tests/controllers/test_uaa_controller.py

```diff
@@ -40,6 +55,8 @@ def setUp(self):
         self.access_token = jwt.encode(payload, self.app.config["UAA_PRIVATE_KEY"], algorithm="RS256")
         self.client = self.app.test_client()
 
+    # user_has_permission
```

Comment:
This looks strange, not sure why linting hasn't picked this up?

Reply:
I think because it's valid as you can have a comment above a function with an extra line.  But you're right, it does look weird so I'll move it closer together